code/jupyter_notebook/.ipynb_checkpoints/keys_and_addresses_python_notebook-checkpoint.ipynb

```diff
@@ -15,7 +15,7 @@
     "\n",
     "    $ pip3 install ethereum\n",
     "    \n",
-    "    $ pip3 install bitcoin\n",
+    "    $ pip3 install Bitcoin\n",
```

Comment:
this one is bitcoin, small b. Package names are case-sensitive

Reply:
Yes, be careful with this kind of mega refactoring.